### PR TITLE
Coding style

### PR DESCRIPTION
**Pull request by mimmi20**
With this PR the coding style shall be changed all over the project, so that is the same everythere.
An automatic test on travis is not enabled, because it needs around 40 minutes there.

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<?php
+$header = <<<EOF
+This file is part of the BrowserDetector package.
```
Comment (asgrim) [inline .php_cs]: Package name wrong

File: .php_cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+(c) Thomas Mueller <mimmi20@live.de>
```
Comment (asgrim) [inline .php_cs]: Copyright should be `(c) Copyright 1998-2016 The Browser Capabilities Project`

File: .php_cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
+<?php
+$header = <<<EOF
```
Comment (asgrim) [inline .php_cs]: In fat, `$header` doesn't seem to be used?

Comment (mimmi20) [inline .php_cs]: this code has been removed now.

File: .php_cs (hunk 2 above)
Comment (mimmi20) [inline .php_cs]: this code has been removed now.

File: .php_cs (hunk 1 above)
Comment (mimmi20) [inline .php_cs]: this code has been removed now.

**Comment by asgrim** [comment]
Conflicts (naturally) - I'll make sure this is the next merge once conflicts are resolved @mimmi20 :) thanks

**Comment by mimmi20** [comment]
@asgrim The conflict is solved.

**Comment by asgrim** [comment]
Thanks! :)

**Pull request merged**